pep-0001.txt

```diff
@@ -603,11 +603,9 @@ The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
 The format of a PEP is specified with a Content-Type header.
-Valid values are ``text/plain`` for plaintext PEPs (see :pep:`9`)
-and ``text/x-rst`` for reStructuredText PEPs (see :pep:`12`).
-All new and active PEPs must use reStructuredText, but for backwards
-compatibility, plain text is currently still the default if no
-Content-Type header is present.
+All PEPs now use reStructuredText (see :pep:`12`),
```

Comment:
Remove now, and the reference to PEP 12. PEP 9 does not follow the reST template in PEP 12, but does use reST.

```suggestion
All PEPs use reStructuredText,
```

Reply:
I removed now "now" and "must" per @warsaw 's suggestion. Unless there's something I'm misunderstanding, it seems rather silly to remove the reference to PEP 12 just because one withdrawn PEP doesn't include the suggested; in truth, many PEPs don't to varying extents, but they still use the standard headers and valid reST syntax (as does PEP 9).

Reply:
Maybe "(see :pep:`12` for the reStructuredText template)"? It didn't feel particularly linked, but equally is rather minor.